handsontable/src/core.js

```diff
@@ -2975,6 +2975,10 @@ export default function Core(rootElement, userSettings, rootInstanceSymbol = fal
    * __Note__: This method does not participate in data transformation. If the visual data of the table is reordered,
    * sorted or trimmed only physical indexes are correct.
    *
+   * __Note__: By design, this method doesn't support [formulas](@/guides/formulas/formula-calculation.md).
```

Comment:
I'd give more details here. Even when FormulasPlugin is enabled, `getSourceData` returns the correct result for all cells except the formula cells.

Something along the lines of:
```
If you use formulas, `getSourceData` might return incorrect values for the cells that contain formulas.
```

Reply:
I agree with @sequba.

Reply:
@sequba @wszymanski ok, would this be ok to you? [650b489](https://github.com/handsontable/handsontable/pull/10353/commits/650b4895bab09c5c9ff9258b5e7b72c6ce316f64)